core/retries-api/src/main/java/software/amazon/awssdk/retries/api/internal/AcquireInitialTokenRequestImpl.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.retries.api.internal;
+
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.retries.api.AcquireInitialTokenRequest;
+import software.amazon.awssdk.utils.Validate;
+
+/**
+ * Implementation of the {@link AcquireInitialTokenRequest} interface.
+ */
+@SdkInternalApi
+public final class AcquireInitialTokenRequestImpl implements AcquireInitialTokenRequest {
```

Comment:
Added this implementation here in the SPI package as decided in a previous review.  See the [decision log here](https://github.com/aws/aws-sdk-java-v2/blob/feature/master/sra-identity-auth/docs/design/core/sra-identity-auth/DecisionLog.md).